hedera-node/hedera-schedule-service-impl/src/test/java/module-info.java

```diff
@@ -11,6 +11,7 @@
     requires com.hedera.node.app.spi.fixtures;
     requires com.swirlds.common;
     requires org.assertj.core;
+    requires com.hedera.node.app.service.token;
```

Comment:
Is the token service needed in the Schedule Service?

Reply:
This is the module-info of the test folder. (I thought, that we decided to do without in unit tests, but anyway...)

Currently it is needed, because the unit tests depend on the functionality in `PreHandleContext`, which requires the `ReadableAccountStore`. This used to be provided by `AccountAccess` which was part of hedera-app-spi. But we are going to change this, because all stores are now accessible to all services and there is no reason to have special treatment of `ReadableAccountStore`.

The proper fix IMO would be to change the unit tests and make them independent of the implementation of `PreHandleContext`. But this needs to be done by somebody who has a better understanding of the business logic.

Reply:
yes, the module-info files should be deleted. Did it some time ago but the PR was never reviewed and ended in merge hell... (https://github.com/hashgraph/hedera-services/pull/4595)

Reply:
Ahh.  I wondered why the test code had module info.  I'll note that for possible inclusion in my next PR.